src/js/add-remove-localstorage.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-classes-per-file */
+
+export default class LocalStorageClass {
+  static getActFromStore() {
+    let activities;
+    if (localStorage.getItem('activities') === null) {
+      activities = [];
+    } else {
+      activities = JSON.parse(localStorage.getItem('activities'));
+    }
+    return activities;
+  }
+
+  static addActToStore(activity) {
+    const activities = LocalStorageClass.getActFromStore();
+    activities.push(activity);
+    localStorage.setItem('activities', JSON.stringify(activities));
+  }
+
+  static removeFromTheStore(description) {
+    const activities = LocalStorageClass.getActFromStore();
+    const filteredArray = activities.filter((action) => action.description !== description);
+
+    filteredArray.forEach((act, indices) => {
+      act.index = indices + 1;
+    });
+    localStorage.setItem('activities', JSON.stringify(filteredArray));
+  }
+}
+
+export class StoreActivity {
+  constructor(index, description, complete) {
+    this.index = index;
+    this.description = description;
+    this.complete = complete;
+  }
+}
```

Comment:
- [ ] Let's make sure that whenever an activity is marked as completed the correct state is stored on local storage.
> All changes to the To Do List should be saved in local storage.

![no-completed](https://user-images.githubusercontent.com/79114233/150185248-6ca81fd9-deec-43e4-a034-2d56210add9d.gif)